pkg/3scale/amp/operator/system_postgresql_options_provider.go

```diff
@@ -187,6 +187,12 @@ func (s *SystemPostgresqlOptionsProvider) podTemplateLabels() map[string]string
 		labels[k] = v
 	}
 
+	if s.apimanager.IsSystemPostgreSQLEnabled() &&
```

Comment:
checking `s.apimanager.Spec.System.DatabaseSpec.PostgreSQL.Labels != nil `. Checking that postgresql is enabled is correct.

Reply:
Addressed in this [commit](https://github.com/3scale/3scale-operator/pull/835/commits/f903bf41523372ad2297594e4efac295a12c0665)